mk/xe-linux-distribution

```diff
@@ -363,6 +363,36 @@ identify_kylin()
     write_to_output "${distro}" "${major}" "${minor}" "$(head -n 1 ${kylin_release})"
 }
 
+
+identify_kylin()
+{
+    kylin_release="$1"
+    local distro
+    local major
+    local minor
+
+    # distro
+    # Kylin Linux Advanced Server release V10 (Sword)
+
+    if [ ! -f "${kylin_release}" ] ; then
+        return 1
+    fi
+
+    eval $(sed -rn \
+            's/^Kylin Linux Advanced Server release V([0-9][0-9]).*$/distro=kylin;major=\1;/gp;' \
```

Comment:
`V([0-9][0-9])` could be `V([0-9]{2})`

Reply:
It's a mistake in the commit, I'v merged all of the commit into one. Thus it will not show this modification anymore. Previous  ly one commit modify this, another commit recover this. I do this modification to test Kylin V10, should not exist.